iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/conf/CommonConfig.java

```diff
@@ -251,13 +253,13 @@ public class CommonConfig {
 
   private int subscriptionSubtaskExecutorMaxThreadNum =
       Math.min(5, Math.max(1, Runtime.getRuntime().availableProcessors() / 2));
-  private int subscriptionMaxTabletsPerPrefetching = 64;
-  private int subscriptionMaxTabletsSizeInBytesPerPrefetching = 8388608; // 8MB
+  private int subscriptionPrefetchBatchMaxDelayInMs = 5000; // 5s
+  private long subscriptionPrefetchBatchMaxSizeInBytes = 80 * MB;
```

Comment:
In pipes, we use 1s and 16MB by default in tablet plain batch. Is it OK to use 5s and 80MB (which are the default settings of tsFile batch) in both batches?

Reply:
Previously overlooked this issue. Now it stays consistent with pipe.